packages/definitions-parser/src/packages.ts

```diff
@@ -181,65 +175,52 @@ function* flattenData(data: ReadonlyMap<string, TypingsVersions>): Iterable<Typi
 
 export type AnyPackage = NotNeededPackage | TypingsData;
 
-interface BaseRaw {
-  /**
-   * The name of the library.
-   *
-   * A human readable version, e.g. it might be "Moment.js" even though `packageName` is "moment".
-   */
-  readonly libraryName: string;
-}
-
 /** Prefer to use `AnyPackage` instead of this. */
 export abstract class PackageBase {
   static compare(a: PackageBase, b: PackageBase): number {
     return a.name.localeCompare(b.name);
   }
 
-  /** Note: for "foo__bar" this is still "foo__bar", not "@foo/bar". */
+  /** The package.json "name" field */
   abstract readonly name: string;
```

Comment:
The meaning of this field is changing; is that intentional? Or maybe this already happened as a part of @andrewbranch's other PR?

Reply:
Hm, reading `NotNeededPackage`, it doesn't have the `@types/` prefix? I am confused :D

Reply:
No, it does. In NotNeededPackage's constructor:

```
this.name = name.startsWith(`@${scopeName}/`) ? name : `@${scopeName}/${name}`;
```

If you pass in a value with `@types/X`, you'll get that; otherwise `@types/` will be prepended.

Reply:
Oh, also, @andrewbranch's PR merged into this one. So it's part of his PR.

Reply:
Right, it's definitely the name with `@types/` now